overlord/ifacestate/helpers.go

```diff
@@ -95,6 +98,62 @@ func (m *InterfaceManager) addSnaps() error {
 	return nil
 }
 
+func (m *InterfaceManager) unbreakTheWorld() error {
+	// XXX: As a special measure to unbreak the world refresh seccomp security
+	// of all the snaps on the system. Later on this should be improved to
+	// either refresh all security backends or to know how to apply stashed
+	// (versioned) security that was preserved by snapd on upgrade to a new
+	// version of itself.
+
+	// Get all the security backends
+	securityBackends := m.repo.Backends()
+
+	// Get all the snap infos
+	snaps, err := snapstate.ActiveInfos(m.state)
```

Comment:
For inactive stuff there's nothing to do ATM: first of all profiles are not versioned (just the current revision can exist) and second of all once activated those profiles will get generated.